tests/Regressions.hs

```diff
@@ -71,6 +76,48 @@ propEqAfterDelete (Keys keys) =
 mapFromKeys :: [Int] -> HM.HashMap Int ()
 mapFromKeys keys = HM.fromList (zip keys (repeat ()))
 
+------------------------------------------------------------------------
+-- Issue #254
+
+data KC = KC Int
```

Comment:
Is `KC` an abbreviation? For what?

Reply:
Is it important that this is a `data` type and not a newtype?

Reply:
KeyCollides. Data vs newtype is irrelevant. I updated the test to say that.